libGitWrap/Commit.cpp

```diff
@@ -347,23 +345,23 @@ namespace Git
 
     DiffList Commit::diffFromParent(Result& result, unsigned int index)
```

Comment:
Yeah, I'll add the docs. And, while I'm on it. I see what else needs documentation :).

Actually the method should be renamed to "diffToParent" - this is what it does. Don't remember why we decided for "from", but it is misleading, don't you think?